cfgrammar/src/lib/yacc/parser.rs

```diff
@@ -334,13 +335,24 @@ impl YaccParser {
     }
 
     fn parse_action(&mut self, i: usize) -> YaccResult<(usize, String)> {
-        match RE_ACTION.find(&self.src[i..]) {
-            Some(m) => {
-                assert!(m.start() == 0 && m.end() > 0);
-                let s = self.src[i + 1..i + m.end() - 1].trim().to_string();
-                Ok((i + m.end(), s))
+        let mut j = i + 1; // Skip opening brace
```

Comment:
I would probably just make `i` mutable in the function signature (i.e. `fn parse_action(&mut self, mut i: usize)` and then there's no point incrementing `+ 1` here. Just `let mut c = 0` and it should handle everything.

Reply:
You mean get rid of `j`?

Reply:
Yep.

Reply:
I don't think I can get rid of `j` since I need both `i` and `j` to slice the result string.

Reply:
Aha, good point! Yes, agreed. Ignore what I said here.